### PR TITLE
Add Etherlite support

### PR DESCRIPTION
**Pull request by andhovesyan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ethereum-multicall",
-  "version": "2.5.0",
+  "name": "ethereum-multicall-etl",
+  "version": "2.5.1",
```
Comment (joshstevens19) [inline package.json:3]: Please remove this package update.

File: package-lock.json [hunk 2]
```diff
@@ -1,8 +1,921 @@
...
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "version": "2.5.1",
+      "license": "ISC",
+      "dependencies": {
+        "@ethersproject/providers": "^5.0.10",
+        "ethers": "^5.0.15"
+      },
```
Comment (joshstevens19) [inline package-lock.json:13]: Please undo this.. your change should only be the files itself. I will update these on the next deployment

**Review by joshstevens19 — CHANGES_REQUESTED** [review]
Thanks for PR dropped a few comments to sort.

File: package.json (hunk 1 above)
Comment (andhovesyan) [reply package.json]: Thanks for a quick reply, all extra changes are removed

**Comment by andhovesyan** [comment]
> Thanks for PR dropped a few comments to sort.

Thanks for a quick reply, all extra changes are removed.
Will be great if you publish this to npm as well, so that I'll be able to delete my package))

**Comment by joshstevens19** [comment]
@andhovesyan will merge it but won’t be able to publish until Monday as away on holiday. Will let you know when it’s published. Thanks a lot for the PR.

**Comment by joshstevens19** [comment]
Sorry about the delay 2.6.0 is now deployed to NPM

**Pull request merged**